tests/test_bundle.py

```diff
@@ -934,13 +934,22 @@ def test_patch(self):
     def test_patch_excessive(self):
         bundle_uuid, bundle_version = self._put_bundle()
         files = self._patch_files(number_of_files=2000)
-        res = self.app.patch(
-            f"/v1/bundles/{bundle_uuid}",
-            headers=get_auth_header(authorized=True),
-            params=dict(version=bundle_version, replica="aws"),
-            json=dict(add_files=files),
-        )
-        self.assertEqual(res.status_code, requests.codes.bad_request)
+        with self.subTest("Bad request is returned when adding > 1000 files in a single request."):
```

Comment:
why does line 936 say 2000, but the tests say 1000?

Reply:
The number of files (2000) needs to be greater than the limit (1000), to trigger the error.

Changed to 1001.